### ISSUE TITLE
Display volunteer count

### ISSUE THREAD

**Issue by tomahawk-pilot** [IEEEKeralaSection/rescuekerala#1030]
Display total registered volunteers count in the home page or sub pages will be goood

**Comment by jerinisready** [comment]
Am working over it

**Comment by tomahawk-pilot** [comment]
https://keralarescue.in/dmodash/ .. has the stats but a volunteers count in the main page would be nice

**Comment by vigneshhari** [comment]
Make sure it's not db intensive, if its on the main page

**Comment by sreenadh** [comment]
The total count is calculating in the below page for pagination purpose. Show the total there.  One more thing, make the search filter "district" mandatory, otherwise, it will count the entire 75600 records from DB which is not necessary.

https://keralarescue.in/volunteerdata

**Comment by jerinisready** [comment]
Is this okey?  Everything is cached! 


![Screenshot from 2019-08-10 13-23-09](https://user-images.githubusercontent.com/19725964/62819372-04a37d80-bb72-11e9-8b30-a6ddda1fc940.png)

**Comment by sreenadh** [comment]
@jerinisready How is the cache invalidated?

**Comment by jerinisready** [comment]
> The total count is calculating in the below page for pagination purpose. Show the total there. One more thing, make the search filter "district" mandatory, otherwise, it will count the entire 75600 records from DB which is not necessary.
> 
> https://keralarescue.in/volunteerdata

what about applying view caching accross this data?

**Comment by jerinisready** [comment]
> ?

with default timeout! should i manually invalidate with updation on creation of records??

**Comment by sreenadh** [comment]
Yes. The cache should be invalidated on every update/creation of records. 

Does showing the count on the home page really useful?

**Comment by jerinisready** [comment]
> Yes. The cache should be invalidated on every update/creation of records.
> 
> Does showing the count on the home page really useful?

@sreenadh  Probably this will take a lot of public interest. what about continuing communication at slack? share handles here @sreenadh @vigneshhari @tomahawk-pilot. so that we can take care of more issues now! including optimization.

**Comment by sreenadh** [comment]
@jerinisready which channel?

**Comment by jerinisready** [comment]
'issue1017'   its a locked channel,

**Comment by jerinisready** [comment]
started a new one issue1030

**Comment by jerinisready** [comment]
started a new one issue1030

**Comment by tomahawk-pilot** [comment]
@syam in slack

**Comment by tomahawk-pilot** [comment]
@sreenadh all we need is a volunteer count somewhere in the main site. just to boost morale and more people will signup

**Comment by vhawk19** [comment]
Is this done? or is work required?

**Comment by jerinisready** [comment]
https://github.com/IEEEKeralaSection/rescuekerala/pull/1036
